src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading (lines 16-25). Also, ensure that you are using 2 spaces for indentation throughout your file to maintain consistent formatting.